src/regl_rendering.ts

```diff
@@ -126,7 +126,7 @@ export class ReglRenderer extends Renderer {
     const { prefs } = this;
     const { transform } = this.zoom;
     const { aes_to_buffer_num, buffer_num_to_variable, variable_to_buffer_num } = this.allocate_aesthetic_buffers();
-    console.log(prefs.arrow_table);
+    // console.log(prefs.arrow_table);
```

Comment:
Feel free to just kill anything that isn't a console.warn or console.error. I would lint them out of existence except there are few cases that they're actually intentional, alas.